app/agents/supervisor.py

```
@@ -0,0 +1,217 @@
+    #                         "responses": active_session.responses or {},
+    #                         "state": active_session.state
+    #                     }
+    #             break
+    #     except Exception as e:
+    #         logger.error(f"Error checking active wizard session: {e}")
+    #     return None
+
+    def decide_next_agent(self, state: ConversationState) -> str:
+        """Decide el próximo agente en el flujo del grafo"""
+
+        current_agent = state.get("current_agent", "supervisor")
```

Ruff (0.12.2)

Local variable `current_agent` is assigned to but never used

Remove assignment to unused variable `current_agent`

(F841)